04/List.js

```diff
@@ -1,8 +1,14 @@
 import React from 'react';
+import {v4 as uuid} from 'uuid';
+
 
 class List extends React.Component {
     render() {
-        return <ul>Lista plików</ul>
+        const {elements} = this.props;
+        const filesList = elements.map(file => {
+            return <li key={uuid()}> name={file.name} size={file.size} content={file.content}</li>
```

Comment:
Jesli używamy tak `uuid()` to nie wiele to daje bo za każdym "renderem" mamy inne k`ey`